openedx/core/djangoapps/programs/tests/test_backpopulate_program_credentials.py

```diff
@@ -40,6 +42,9 @@ def setUp(self):
         self.alice = UserFactory()
         self.bob = UserFactory()
 
+        # We need CourseOverview instances to exist for the test courses
+        CourseOverviewFactory(id=CourseKey.from_string(self.course_run_key))
+        CourseOverviewFactory(id=CourseKey.from_string(self.alternate_course_run_key))
```

Comment:
Could maybe add a test confirming that a course without a backing course overview is appropriately not included?

Reply:
done